sections/futures/Trade/DepositMarginModal.tsx

```diff
@@ -156,61 +129,62 @@ const DepositMarginModal: React.FC<DepositMarginModalProps> = ({
 				onChange={(_, v) => setAmount(v)}
 				right={<MaxButton onClick={handleSetMax}>Max</MaxButton>}
 			/>
+
 			<MinimumAmountDisclaimer>
-				Note: Placing an order requires a minimum deposit of 50 sUSD.
+				A $50 margin minimum is required to open a position.
```

Comment:
We've decided to start using `i18n` again so we should really start refactoring in the `en.json` again when we come across these things.

Reply:
True. I can add them as part of this PR, or make a separate PR for them (since there are a few places where we're hardcoding text, that are not covered in the scope of this PR)

Reply:
Add these for this one that way we do some refactoring as we go